src/triage/component/results_schema/nuke_triage.sql

```diff
@@ -0,0 +1,48 @@
+/*
+  Function for wiping all triage tables, schemas and indexes.
+Useful  when ou try to start clean (again) and try to recover some
+previous errors
+ */
+create or replace function nuke_triage()
+    returns text as $result$
+
+    declare
+    result text;
+    query text;
+
+    begin
+
+    execute 'drop schema if exists triage_metadata cascade';
+    raise notice 'triage_metadata deleted';
+    execute 'drop schema if exists features cascade';
+    raise notice 'features deleted';
+    execute 'drop schema if exists train_results cascade';
+    raise notice 'train_results deleted';
+    execute 'drop schema if exists test_results cascade';
+    raise notice 'test_results deleted';
+
+    execute 'drop table if exists results_schema_versions';
+    raise notice 'results_schema_versions deleted';
+
+    execute 'drop type if exists experimentrunstatus';
+    raise notice 'experimentrunstatus type deleted';
+
+select into query
+    string_agg(
+        format('drop table %I cascade;', tablename), E'\n'
+        )
+from   pg_tables
+where  tablename ~ 'cohort_|labels_|ranks_';
```

Comment:
feels like this could potentially be quite aggressive, especially since it doesn't restrict to a schema. Any want to better identify the triage-specific tables? I could easily see someone having a table named `labels_{something}` somewhere in their database that gets killed by this unexpectedly.

Reply:
`triage` currently puts `labels` `cohorts` and `ranks` tables in the `public` schema. This is a very BAD practice. Almost an anti pattern. I am moving those tables into some new triage related schema, and I will update the `nuke_triage` function accordingly

Reply:
yeah, that's always bothered me, too -- a `triage_internals` schema or something like that might be good. In a separate PR, I assume?